src/reusable-blocks/class-reusable-blocks-basket-handler.php

```diff
@@ -28,7 +28,7 @@ public function add_blocks( array $data ) {
 		$blocks        = $this->get_blocks_from_post_elements( $post_elements );
 
 		$blocks = $blocks->map( [ $this, 'select_target_langs' ] )
-			->filter( function( $block ) { return ! empty( $block['target_langs'] ); } )
+			->filter( function( $block ) { return ! empty( $block->target_langs ); } )
```

Comment:
@strategio You can use reject instead of the not empty. It reads nicer to me.
```php
->reject( function( $block ) { return empty( $block->target_langs ); } )
```

Reply:
Good idea @brucepearson! New habits are coming :)